### PR TITLE
implemented link from index to show page, no styling

### PR DESCRIPTION
**Pull request by ArtyG-10**

### CODE REVIEW

File: app/controllers/apartments_controller.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ def index
     apartments = Apartment.all
     render json:apartments
   end
+  def create
+    apartment = Apartment.create(name: params[:name], age: params[:age], enjoys: params[:enjoys])
```
Comment (sjproctor) [inline app/controllers/apartments_controller.rb]: You need strong params here. And I think some cat tinder code snuck in here.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Controller needs a little love.

File: app/controllers/apartments_controller.rb (hunk 1 above)
Comment (ArtyG-10) [reply app/controllers/apartments_controller.rb]: Okay thanks! I should have commented that out since I am just now starting my create functionality.

Comment (ArtyG-10) [reply app/controllers/apartments_controller.rb]: pushed changes

**Review by sjproctor — APPROVED** [review]
Looks good!

**Pull request merged**